.github/workflows/bake.yaml

```
@@ -1,74 +1,128 @@
+          echo "::notice title=Target Registries::${TARGETS}"
+
+      - name: 🧰 Set up QEMU
         uses: docker/setup-qemu-action@v3
-      - name: set up Docker Buildx
+
+      - name: 🧰 Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: login to ECR (Private)
+      - name: 🔐 Log in — AWS ECR (private)
         if: ${{ fromJSON(inputs.private-ecr) || fromJSON(inputs.gcr) }}
```

actionlint (1.7.12)

[error] 1st argument of function call is not assignable. "bool" cannot be assigned to "string". called function type is "fromJSON(string) -> any"

(expression)